Client/Client.Core/Client.Core.Static/Client.hpp

```diff
@@ -162,6 +162,8 @@ class Client
     virtual void onChannelSubscribingListAnswer(const std::vector<uint64_t> subscribingChannelList);
     /// Channel delete Answer handler
     virtual void onChannelDeleteAnswer(Utility::ChannelDeleteCode channelDeleteCode);
+    /// Channel edit Answer handler
+    virtual void onEditMessageAnswer(Utility::EditingMessageCodes reactionState) const;
```

Comment:
Why const? I mean, others are not, but this is. Especially this should be const or others should be const however it doesn't happen

Reply:
Agree with you. We should need to changes others methods to const.